app/hotels/src/map/allHotels/HotelSwipeList.js

```diff
@@ -89,49 +59,64 @@ class HotelSwipeList extends React.Component<Props, State> {
   render = () => {
     const { data, onSnapToItem } = this.props;
 
-    const child = (
-      <BottomSheet openHeight={openHeight} closedHeight={closedHeight}>
-        <BottomSheetHandle />
-        {data.length ? (
-          <React.Fragment>
-            <View style={styles.sliderWrapper}>
-              <Carousel
-                ref={this.storeRef}
-                data={data}
-                renderItem={this.renderItem}
-                sliderWidth={Device.DEVICE_THRESHOLD}
-                itemWidth={CARD_ITEM_WIDTH}
-                inactiveSlideScale={1}
-                inactiveSlideOpacity={0.5}
-                decelerationRate="fast"
-                activeSlideAlignment="start"
-                containerCustomStyle={styles.slider}
-                removeClippedSubviews={false}
-                onSnapToItem={onSnapToItem}
-                useScrollView={true}
-              />
-            </View>
-            <Address address={this.getSelectedAddress()} />
-          </React.Fragment>
-        ) : (
-          <View style={styles.noResultsContainer}>
-            <Translation id="hotels.map.no_results" />
-          </View>
-        )}
-      </BottomSheet>
-    );
-
     return (
-      <AdaptableLayout
-        renderOnWide={
-          <View style={[styles.fullWidth, styles.wide]}>{child}</View>
-        }
-        renderOnNarrow={<View style={styles.fullWidth}>{child}</View>}
-      />
+      <View style={styles.container}>
+        <BottomSheet openHeight={openHeight} closedHeight={closedHeight}>
+          <BottomSheetHandle />
+          {data.length ? (
```

Comment:
If it is zero, then you'd have no results too, right?

Reply:
that is right

Reply:
Since this is a ternary it will render the error if data.length is null, undefined or 0. 

if this was {data.length && <.../>} it would cause crash on android if data.length was 0